.travis.yml

```diff
@@ -3,18 +3,11 @@ env:
   - VERIFY_USE_PUBLIC_BINARIES=true
 jdk:
   - oraclejdk8
-  - oraclejdk9
-  - oraclejdk10
   - openjdk8
-  - openjdk9
   - openjdk10
```

Comment:
Do we really need openjdk10 here? Since we're aiming to migrate straight to 11, we could free up more executors by not including it.